custom_components/hacs/helpers/functions/file_etag.py

```diff
@@ -2,20 +2,19 @@
 import os
 
 from custom_components.hacs.share import get_hacs
-from fnvhash import fnv1a_32
 from pathlib import Path
 
 
 def get_etag(path) -> bool:
     file_path = Path(path)
     if not file_path.exists():
         return None
-    return fnv1a_32(file_path.read_bytes())
+    return hash(file_path.read_bytes())
 
 
 async def async_get_etag(path) -> bool:
     hass = get_hacs().hass
-    fnv = await hass.async_add_executor_job(get_etag, path)
-    if fnv is None:
+    etag = await hass.async_add_executor_job(get_etag, path)
+    if etag is None:
         return None
-    return str(hex(fnv))
+    return '"' + str(hex(etag)) + '"'
```

Comment:
Why is this needed? `str()`  makes it a string right?

Reply:
Firefox actually wants the whole string in quotes

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag